staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/controller.go

```diff
@@ -191,7 +193,7 @@ func buildVersionSpecs(crd *apiextensions.CustomResourceDefinition, oldSpecs map
 		if !v.Served {
 			continue
 		}
-		spec, err := BuildSwagger(crd, v.Name)
+		spec, err := builder.BuildSwagger(crd, v.Name, true)
```

Comment:
when the v3 builder is implemented, we call a `builder.BuildSwagger(crd, v.Name, false)` here and update by `c.openAPIV3Service.UpdateSpec`, is that true?

Reply:
there will be more necessary to do that. This is only the small CRD builder.